output/cmf.eslint.txt

```diff
@@ -5,7 +5,7 @@
 The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.
 
 /home/travis/build/Talend/ui/packages/cmf/src/cmfConnect.js
-  151:4  warning  Unexpected console statement  no-console
+  152:4  warning  Unexpected console statement  no-console
```

Comment:
Maybe use invariant instead

Reply:
this one is for a deprecation warning, invariant throw an error, so it block the app in dev.